src/bone.js

```diff
@@ -1557,7 +1557,12 @@ class Bone {
     if (columns.length === 0) {
       await driver.createTable(table, attributes);
     } else {
-      await driver.alterTable(table, compare(attributes, columns));
+      if (force) {
+        await driver.dropTable(table);
+        await driver.createTable(table, attributes);
+      } else if (alter){
+        await driver.alterTable(table, compare(attributes, columns));
+      }
```

Comment:
need a default `else` branch to output some warning, informing user that some models couldn't be synchronized because neither force or alter is enabled.